### DISCUSSION TITLE
Can't update a Sonoff POW r2

### DISCUSSION THREAD

**Discussion by Avatar1976** [arendst/Tasmota#22604] [Q&A]
The Sonoff POW r2 I have been running Tasmota on for years (since around 2021), won't update via ota methods including the Tamoadmin in Home Assistant.
I have attached my information page here, I get messages like insufficient space etc. 
![Screenshot_20241208_155842_Edge](https://github.com/user-attachments/assets/53f5010c-bd36-4ecd-94b9-e209136c0720)
Is there a trick to redoing FS to support newer versions or something I am not aware of, ota methods preferable as the switch is in a difficult place to reach.

**Comment by sfromis** [comment]
Did you make sure to **always** use the `.bin.gz` variant when doing OTA upgrading?
Preferable to *not* download first, as Tasmota can automatically do an extra minimal step if/when relevant.

I'm seeing no trouble upgrading Sonoff Pow R2 with current Tasmota firmware binaries.
